### PR TITLE
Update README to point to Github instead of BitBucket - fixes #242

### LINKED ISSUE #242: Update Readme to Point to Github Issues instead of BitBucket Issues
Per #7, github is now the preferred location to report issues, but README.md still points to still points to the bitbucket issues in https://github.com/xerial/sqlite-jdbc/blame/9833b1174b629b5a8956e93e04a88468c95081de/README.md#L32

Should that point to github at this point?

### PR DESCRIPTION
**Pull request by sualeh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -103,7 +103,7 @@
           <goals>deploy</goals>
           <pushChanges>false</pushChanges>
           <localCheckout>true</localCheckout>
-          <connectionUrl>scm:hg:ssh://hg@bitbucket.org/xerial/sqlite-jdbc</connectionUrl>
+          <connectionUrl>cm:git:git://github.com/xerial/sqlite-jdbc.git</connectionUrl>
```
Comment (PromanSEW) [inline pom.xml]: Maybe`scm:git:git://github.com/xerial/sqlite-jdbc.git`?
Typo: first `s` is missing

Comment (sualeh) [inline pom.xml]: Thanks for finding that. I will fix and re-submit.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ SQLite JDBC Driver [![Build Status](https://travis-ci.org/xerial/sqlite-jdbc.svg
 ==================
 SQLite JDBC, developed by [Taro L. Saito](http://www.xerial.org/leo), is a library for accessing and creating [SQLite](http://sqlite.org) database files in Java.
 
-Our SQLiteJDBC library requires no configuration since native libraries for major OSs, including Windows, Mac OS X, Linux etc., are assembled into a single JAR (Java Archive) file. The usage is quite simple; [download](https://bitbucket.org/xerial/sqlite-jdbc/downloads)
+Our SQLiteJDBC library requires no configuration since native libraries for major OSs, including Windows, Mac OS X, Linux etc., are assembled into a single JAR (Java Archive) file. The usage is quite simple; [download](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22sqlite-jdbc%22%20g%3Aorg.xerial)
```
Comment (xerial) [inline README.md]: Could you keep the URL as is? We are still using bitbucket for downloading.

File: README.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ Usage
 ============
 SQLite JDBC is a library for accessing SQLite databases through the JDBC API. For the general usage of JDBC, see [JDBC Tutorial](http://docs.oracle.com/javase/tutorial/jdbc/index.html) or [Oracle JDBC Documentation](http://www.oracle.com/technetwork/java/javase/tech/index-jsp-136101.html).
 
-1.  Download sqlite-jdbc-(VERSION).jar from the [download page](https://bitbucket.org/xerial/sqlite-jdbc/downloads) (or by using [Maven](#using-sqlitejdbc-with-maven2))
+1.  Download sqlite-jdbc-(VERSION).jar from the [download page](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22sqlite-jdbc%22%20g%3Aorg.xerial) (or by using [Maven](#using-sqlitejdbc-with-maven2))
```
Comment (xerial) [inline README.md]: Preserve the bitbucket url.

File: README.md (hunk 2 above)
Comment (sualeh) [reply README.md]: Will do

File: README.md (hunk 3 above)
Comment (sualeh) [reply README.md]: Will do

**Pull request closed without merging**